packages/web-components/fast-components/src/horizontal-scroll/fixtures/horizontal-scroll.html

```diff
@@ -0,0 +1,301 @@
+<h1>Horizontal scroll</h1>
+
+<style type="text/css">
+    fast-card {
+        width: 120px;
+        height: 200px;
+        margin: 5px;
```

Comment:
@robarbms @radium-v I'm just seeing that the margin is coming from elements themselves. Could we leverage the slotted selector here to apply this via CSS Property to any slotted item - with exception of the *last*?

Reply:
Added --scroll-item-spacing with a default of 5px.